src/get-log-level.ts

```diff
@@ -0,0 +1,9 @@
+// re-map the log level option to match the bugsnag-cli log level
+const getLogLevel = (logLevel: string | undefined) => {
```

Comment:
@joshedney I've added a helper to ensure the provided log level matches the level in the [cli](https://docs.bugsnag.com/build-integrations/bugsnag-cli/upload-js/#options-log-level)